scripts/variantstore/beta_docs/gvs-quickstart.md

```diff
@@ -63,7 +64,7 @@ If you already have a GCP project that you would like to use to test the GVS wor
 
 ### Step 4. Create a BigQuery dataset
 
-BigQuery datasets store the tables created during the execution of the GVS workflow. A new dataset should be created for each callset you want to create using the workflow. Samples can be added to BiqQuery datasets cumulatively and any samples in the dataset will be included in the extracted callset. 
+BigQuery datasets store the tables created during the execution of the GVS workflow. A new dataset should be created for each callset you want to create using the workflow. 
 
 Create a dataset in BigQuery inside the GCP project you created in Step 3 (above) by following the instructions in the Google Cloud documentation article, [Creating datasets](https://cloud.google.com/bigquery/docs/datasets#create-dataset). We recommend that you select "Multi-region" for the dataset's "Location type" in order to avoid more restrictive Google quotas.
```

Comment:
Uhhhhh this is the only place where we actually recommend multi-region.  Given that it's much more expensive than it used to be now, do you stand by that recommendation?

Reply:
I had the same thought when I read this. We clarified with Google that there are no additional multi-region charges when dealing with BigQuery datasets. The recent multi region pricing changes are only for cloud storage.